portfolio/src/main/webapp/script.js

```diff
@@ -12,9 +12,16 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
- */
+// jQuery for form submission
+$("#comment-form").submit(function(event){
+	event.preventDefault();
+	var postUrl = "/comments";
+	var formData = $(this).serialize();
+
+	$.post(postUrl, formData, function() {
+	  loadComments();
```

Comment:
I think for a good UX, you need to clear the text field too.

Reply:
Oh yeah! I wanted to implement this as well, thanks for reminding me.